nemo/collections/asr/models/ctc_models.py

```diff
@@ -598,14 +599,14 @@ def training_step(self, batch, batch_nb):
         )
 
         if (batch_nb + 1) % log_every_n_steps == 0:
-            self._wer.update(
+            self.wer.update(
                 predictions=log_probs,
                 targets=transcript,
-                target_lengths=transcript_len,
+                targets_lengths=transcript_len,
```

Comment:
Is it necessary to update target_lengths to targets_lengths ?

Reply:
Nah but we're a bit too inconsistent on how we're doing plurals with variables so this seemed the lesser of multiple evils. Else...

```
predictionS
predictionS_lengths
targetS
target_lengths
```

Reply:
Lol time for a fix then.